exchanges/bybit/bybit.go

```diff
@@ -8,740 +8,2348 @@ import (
+func (by *Bybit) GetHistoricalValatility(ctx context.Context, category, baseCoin string, period int64, startTime, endTime time.Time) ([]HistoricVolatility, error) {
+	if category == "" {
+		return nil, errCategoryNotSet
+	} else if category != cOption {
+		return nil, fmt.Errorf("%w, category: %s", errInvalidCategory, category)
+	}
+	params := url.Values{}
+	params.Set("category", category)
+	if baseCoin != "" {
+		params.Set("baseCoin", baseCoin)
+	}
+	if period > 0 {
+		params.Set("period", strconv.FormatInt(period, 10))
+	}
+	var err error
+	if err = common.StartEndTimeCheck(startTime, endTime); !(startTime.IsZero() && endTime.IsZero()) && err != nil {
+		return nil, err
+	} else if !startTime.IsZero() {
+		params.Set("startTime", strconv.FormatInt(startTime.UnixMilli(), 10))
+		params.Set("endTime", strconv.FormatInt(endTime.UnixMilli(), 10))
+	}
+	var resp []HistoricVolatility
+	return resp, by.SendHTTPRequest(ctx, exchange.RestSpot, common.EncodeURLValues("market/historical-volatility", params), defaultEPL, &resp)
+}
+
+// GetInsurance retrieves insurance pool data (BTC/USDT/USDC etc). The data is updated every 24 hours.
+func (by *Bybit) GetInsurance(ctx context.Context, coin string) (*InsuranceHistory, error) {
+	params := url.Values{}
+	if coin != "" {
+		params.Set("coin", coin)
+	}
+	var resp *InsuranceHistory
+	return resp, by.SendHTTPRequest(ctx, exchange.RestSpot, common.EncodeURLValues("market/insurance", params), defaultEPL, &resp)
+}
+
+// GetRiskLimit retrieves risk limit history
+func (by *Bybit) GetRiskLimit(ctx context.Context, category, symbol string) (*RiskLimitHistory, error) {
+	if category == "" {
+		return nil, errCategoryNotSet
+	} else if category != cLinear && category != cInverse {
+		return nil, fmt.Errorf("%w, category: %s", errInvalidCategory, category)
+	}
+	params := url.Values{}
+	params.Set("category", category)
+	if symbol != "" {
+		params.Set("symbol", symbol)
+	}
+	var resp *RiskLimitHistory
+	return resp, by.SendHTTPRequest(ctx, exchange.RestSpot, common.EncodeURLValues("market/risk-limit", params), defaultEPL, &resp)
+}
+
+// GetDeliveryPrice retrieves delivery price.
+func (by *Bybit) GetDeliveryPrice(ctx context.Context, category, symbol, baseCoin, cursor string, limit int64) (*DeliveryPrice, error) {
+	if category == "" {
+		return nil, errCategoryNotSet
+	} else if category != cLinear && category != cInverse && category != cOption {
+		return nil, fmt.Errorf("%w, category: %s", errInvalidCategory, category)
+	}
+	params := url.Values{}
+	params.Set("category", category)
+	if symbol != "" {
+		params.Set("symbol", symbol)
+	}
+	if baseCoin != "" {
+		params.Set("baseCoin", baseCoin)
+	}
+	if limit > 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+	if cursor != "" {
+		params.Set("cursor", cursor)
+	}
+	var resp *DeliveryPrice
+	return resp, by.SendHTTPRequest(ctx, exchange.RestSpot, common.EncodeURLValues("market/delivery-price", params), defaultEPL, &resp)
+}
+
```

Comment:
Looks like you're missing endpoint
https://bybit-exchange.github.io/docs/v5/market/long-short-ratio